RefugeeLand.Core.Api.Tests.Unit/Foundations/RefugeeServiceTests.Validations.Add.cs

```diff
@@ -26,18 +27,22 @@ public async Task ShouldThrowValidationExceptionOnAddIfRefugeeIsNullAndLogItAsyn
             ValueTask<Refugee> addRefugeeTask =
                 this.refugeeService.AddRefugeeAsync(nullRefugee);
 
+            RefugeeValidationException actualRefugeeValidationException =
+                await Assert.ThrowsAsync<RefugeeValidationException>(
+                    addRefugeeTask.AsTask);
+
             // then
-            await Assert.ThrowsAsync<RefugeeValidationException>(() =>
-                addRefugeeTask.AsTask());
+            actualRefugeeValidationException.Should().BeEquivalentTo(
+                expectedRefugeeValidationException);
 
             this.loggingBrokerMock.Verify(broker =>
-                broker.LogError(It.Is(SameExceptionAs(
-                    expectedRefugeeValidationException))),
-                        Times.Once);
+                    broker.LogError(It.Is(SameExceptionAs(
+                        expectedRefugeeValidationException))),
+                            Times.Once);
 
             this.storageBrokerMock.Verify(broker =>
-                broker.InsertRefugeeAsync(nullRefugee),
-                    Times.Never);
+                    broker.InsertRefugeeAsync(nullRefugee),
+                        Times.Never);
```

Comment:
@Spectralgo I am a bit confused about the tabbing here. Do you have two tabs. If so please remove one. But, I like the rest ; )

Reply:
done